README.md

```diff
@@ -0,0 +1,6 @@
+# Framework
+
+This repository is a playground for creating automation testing framework. Every developer proposes a framework and features, so that others know about it and also possibly can contribute to one. 
```

Comment:
Why did you write this? This is not a good practice...!!!

Reply:
Actually I should have write more detailed document. Did not get enough time for this. However, hopefully will write some details in future.
Thanks for noticing this.